client/src/pages/UploadButton.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { makeStyles, Button, Typography, Grid } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    "font-family" : theme.fontFamily,
+  },
+  button: {
+  	"background-color": theme.secondary,
+  	"color": theme.bgcolor,
+  	"border-radius": "0",
+  },
+  buttonContainer: {
+  	"width": "10%",
+  	"margin-left": "45%", 
```

Comment:
this is good, and is valid CSS, but with react you can use camelCase to follow better react conventions :) 
so in this case, this would be: `marginLeft: "45%"`

also using percentages like this is risky! i haven't tested this out, and this may not be wrong but keep in mind diff browsers and resolutions ;) this just looks a little alarming because its quite specific percentages (rather than common ones like 50% or 100%)

Reply:
Oh I had no idea you could do that in camelCase! I'll definitely change that up :) 
Hmm about the percentages, maybe we can hop on a call about that? I would really appreciate your feedback on how I can do better there!